<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsRangeConfigDlgBase</class>
 <widget class="QWidget" name="QgsRangeConfigDlgBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="rangeLabel">
     <property name="text">
      <string>Allows setting of numeric values from a specified range. The edit widget can be either a slider or a spin box.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QgsCollapsibleGroupBox" name="groupBox">
     <property name="title">
      <string>Advanced Options</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="suffixLabel">
        <property name="text">
         <string>Suffix</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="suffixLineEdit">
        <property name="placeholderText">
         <string>Inactive</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="precisionLabel">
        <property name="text">
         <string>Precision</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="precisionSpinBox">
        <property name="toolTip">
         <string>Number of decimal places</string>
        </property>
        <property name="value">
         <number>4</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QComboBox" name="rangeWidget"/>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="valuesLabel">
     <property name="text">
      <string>Local minimum/maximum = 0/0</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QStackedWidget" name="rangeStackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="intPage">
      <layout class="QFormLayout" name="formLayout">
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="minimumLabel">
         <property name="text">
          <string>Minimum</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="minimumSpinBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="maximumLabel">
         <property name="text">
          <string>Maximum</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="maximumSpinBox"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="stepLabel">
         <property name="text">
          <string>Step</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="stepSpinBox"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="doublePage">
      <layout class="QFormLayout" name="formLayout_2">
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item row="2" column="0">
        <widget class="QLabel" name="stepLabel_2">
         <property name="text">
          <string>Step</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="maximumLabel_2">
         <property name="text">
          <string>Maximum</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="minimumLabel_2">
         <property name="text">
          <string>Minimum</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="minimumDoubleSpinBox"/>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="maximumDoubleSpinBox"/>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="stepDoubleSpinBox">
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="allowNullCheckBox">
     <property name="text">
      <string>Allow NULL</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>rangeWidget</tabstop>
  <tabstop>minimumSpinBox</tabstop>
  <tabstop>maximumSpinBox</tabstop>
  <tabstop>stepSpinBox</tabstop>
  <tabstop>minimumDoubleSpinBox</tabstop>
  <tabstop>maximumDoubleSpinBox</tabstop>
  <tabstop>stepDoubleSpinBox</tabstop>
  <tabstop>allowNullCheckBox</tabstop>
  <tabstop>groupBox</tabstop>
  <tabstop>suffixLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
